certbot/storage.py

```diff
@@ -810,6 +810,19 @@ def new_lineage(cls, lineagename, cert, privkey, chain, cli_config):
             logger.debug("Writing full chain to %s.", target["fullchain"])
             f.write(cert + chain)
 
+        # Write a README file to the live directory
+        readme_path = os.path.join(live_dir, "README")
+        with open(readme_path, "w") as f:
+            logger.debug("Writing README to %s.", readme_path)
+            f.write("The files in this directory are symlinks to the latest "
+                    "versions of your keys and certificates.\n\n"
+                    "`privkey.pem` is the private key for your certificate.\n"
+                    "`fullchain.pem` contains all the information your certificate needs, and "
+                    "is likely the certificate file you will need.\n"
+                    "`chain.pem` is used for OCSP stapling in Nginx >=1.3.7.\n\n"
+                    "For more information, see the Certbot User Guide "
+                    "at https://certbot.eff.org/docs/using.html.\n")
```

Comment:
It might be helpful to break this text out into a variable that's set elsewhere so that it will be slightly easier to find and modify.

I'd propose something like this:

```
The files in this directory were created by the Certbot software, which may also be able to renew this certificate automatically. These files are symbolic links to the latest versions of your key and certificate data and are automatically updated when the certificate is renewed. These files, rather than the link targets, should be referred to in server configurations.

privkey.pem  is the private key for your certificate.

fullchain.pem  contains the certificate together with the associated chain certificate, which is needed by some software that connects to your site in order to confirm that the certificate is valid. This is likely the certificate file that you should use with server software.

chain.pem  contains only the chain certificate. This is used in specialized circumstances, such as for HTTPS with Apache versions before 2.4, and OCSP stapling in Nginx versions starting with 1.3.7.

cert.pem  contains only the certificate, which is not sufficient by itself and should not be used in most software. It can be used in specialized circumstances together with chain.pem, such as in Apache versions before 2.4. Most users need to use fullchain.pem instead of cert.pem.

If you have OpenSSL installed on this system, you can see detailed information about the current certificate's content by running

    openssl x509 -in cert.pem -text -noout

For more information, please see the Certbot User Guide at https://certbot.eff.org/docs/using.html; assistance may also be available from the Let's Encrypt Community Forum at https://community.letsencrypt.org/.
```